src/compat/dbtype_def.h

```diff
@@ -1010,6 +1010,7 @@ extern "C"
       const char *buf;
       int compressed_size;
       char *compressed_buf;
+      int length;		//Only Use for group_concat() now
```

Comment:
It is trivial.
I want it to be a comment format similar to the surroundings.
```suggestion
      int length;		/* Only Use for group_concat() now */
```